cmd/flexible-informer-gen/generators/informer.go

```diff
@@ -182,7 +188,12 @@ func NewFiltered$.type|public$Informer(client $.clientSetInterface|raw$$if .name
 				if tweakListOptions != nil {
 					tweakListOptions(&options)
 				}
-				return client.$.group$$.version$().$.type|publicPlural$($if .namespaced$namespace$end$).Watch(context.TODO(), options)
+				watchObj, err := client.$.group$$.version$().$.type|publicPlural$($if .namespaced$namespace$end$).Watch(context.TODO(), options)
+				if $.isNotFound$(err) {
+					logs.warnf("The version of Istio CRD is not matched with Aspen Mesh", logs.ErrorField(err))
```

Comment:
Isn't the code generator usable for non-istio informers as well?  I think the "isNotFound" check might make sense (now the code generator is even _more_ flexible!), but the log message is really specific (talking about istio and aspen mesh)

Reply:
Yeah and also not clear on what happens now.

I'd suggest:
```
The CRD for {group/kind} is {version}, which is too old.  All {kind}s will be ignored.
```

Reply:
Should we explicitly say 'too old'? That should be the common reason but what if the version doesn't exist

Reply:
Good point, I'll leave it to you to edit, as long as the error message tells users what will happen I am good.